library/common.h

```diff
@@ -177,8 +174,19 @@ static inline const unsigned char *mbedtls_buffer_offset_const(
  * \param   a Pointer to input (buffer of at least \p n bytes)
  * \param   b Pointer to input (buffer of at least \p n bytes)
  * \param   n Number of bytes to process.
+ *
+ * \note      Depending on the situation, it may be faster to use either mbedtls_xor or
+ *            mbedtls_xor_no_simd (these are functionally equivalent).
```

Comment:
If we put brackets after the function name (e.g. `mbedtls_xor()`, `mbedtls_xor_no_simd()`), [Doxygen will add links](https://github.com/Mbed-TLS/mbedtls/pull/8433#discussion_r1375915981) (several instances in this PR).

Not sure if this works without the parentheses.